docs/install-mlrun-ce/kubernetes-install.md

```diff
@@ -34,28 +28,12 @@ instructions](https://kubernetes.io/docs/tasks/tools/install-kubectl/) for more
   - 8Gi
   - Set a default storage class for the kubernetes cluster, in order for the pods to have persistent storage. See the [Kubernetes documentation](https://kubernetes.io/docs/concepts/storage/storage-classes/#storageclass-objects) for more information.
 - RAM: A minimum of 8Gi is required for running all the initial MLRun components. The amount of RAM required for running MLRun jobs depends on the job's requirements.
+- Please review the [MLRun CE Installation Notes](./mlrun-ce-installation-notes.md) for any additional installation steps you may need to consider.
```

Comment:
- Review the [MLRun CE Installation Notes](./mlrun-ce-installation-notes.md) for any additional installation steps you may need to consider.